public/styles.css

```
@@ -174,6 +174,11 @@
   font-size: 60px;
 }
 
+.divider {
+  border-top: 1.5px solid #bc1717;
+  margin-top: 10px 0;
```

margin-top only accept a single value.
```suggestion
  margin-top: 10px;
```